domain-models-runtime/src/main/java/org/folio/rest/security/AES.java

```diff
@@ -1,23 +1,20 @@
 package org.folio.rest.security;
 
-
-import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
 import java.util.Base64;
-
-import javax.crypto.BadPaddingException;
 import javax.crypto.Cipher;
-import javax.crypto.IllegalBlockSizeException;
 import javax.crypto.KeyGenerator;
-import javax.crypto.NoSuchPaddingException;
 import javax.crypto.SecretKey;
 import javax.crypto.spec.SecretKeySpec;
 
-
 public class AES {
 
   private static String secretKey = null;
 
+  private AES() {
+    throw new UnsupportedOperationException("Cannot instantiate AES class");
+  }
```

Comment:
Can you add test coverage using https://github.com/folio-org/raml-module-builder/blob/v31.1.0/testing/src/main/java/org/folio/rest/testing/UtilityClassTester.java#L52-L58 , example: https://github.com/folio-org/raml-module-builder/blob/087e899fd2dc12758da09462e52844431c2b30ab/domain-models-runtime/src/test/java/org/folio/rest/tools/utils/TenantToolTest.java#L16

Reply:
Neat.. Should do that for Okapi util classes too.